scholarx/2023/index.html

```diff
@@ -167,11 +167,11 @@ <h1 class="display-3 text-center mb-5">Program Timeline</h1>
           <div class="row justify-content-center justify-content-md-end">
             <div class="col-9">
               <h3 class="mb-2 mt-4 text-onelive">Be a mentor for ScholarX23</h3>
-              <p>
+              <p class="text-justify">
                 Do you consider yourself a qualified professional with a knack for mentoring? 
                 Now you can share your knowledge with budding young talent of Sri Lanka.
-                Reach us via <a
-                href="mailto:sustainableedufoundation@gmail.com">sustainableedufoundation@gmail.com</a>
+                Reach us via <br>
```

Comment:
Can you explain why did you add a line brake here? @RamithaHeshan33

Reply:
Without line break the preview is like this.

![Screenshot 2024-04-25 222140](https://github.com/sef-global/sef-site/assets/142712365/aa1b5f99-b542-45f2-a755-6c4bc0c40000)

That's why I use line break @YoshithaRathnayake

Reply:
Ah ok @RamithaHeshan33